src/main/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStore.kt

```diff
@@ -23,6 +23,7 @@ class ParticipantProjectSpeciesStore(
     private val dslContext: DSLContext,
     private val participantProjectSpeciesDao: ParticipantProjectSpeciesDao,
     private val projectsDao: ProjectsDao,
+    private val submissionStore: SubmissionStore,
```

Comment:
I normally try to avoid having store classes depend on each other both to keep the layering of components more consistent and to reduce the risk of dependency cycles.

There are two approaches the server uses to avoid this kind of layering violation:

1. Introduce a service class that depends on both store classes and calls into both.
2. Publish an application event on one side and handle it on the other.

Reply:
Makes sense, I will make a service class, thanks.